Makefile

```diff
@@ -0,0 +1,90 @@
+SHELL := /bin/bash
+
+# Replace these with the name and domain of your extension!
+NAME     := bring-out-submenu-of-power-off-logout
+DOMAIN   := pratap.fastmail.fm
+ZIP_NAME := $(NAME)@$(DOMAIN).zip
+
+# Some of the recipes below depend on some of these files.
+JS_FILES       = $(shell find -type f -and \( -name "*.js" \))
+ICONS          = $(shell find -maxdepth 1 -type f -and \( -name "*.svg" \))
+UI_FILES       = $(shell find -type f -and \( -name "*.ui" \))
+LOCALES_PO     = $(wildcard po/*.po)
+LOCALES_MO     = $(patsubst po/%.po,locale/%/LC_MESSAGES/$(NAME).mo,$(LOCALES_PO))
+
+# These files will be included in the extension zip file.
+ZIP_CONTENT = $(JS_FILES) $(ICONS) $(LOCALES_MO) \
+              schemas/* schemas/gschemas.compiled metadata.json
```

Comment:
Hi @QuentiumYT line num 28, install command will automatically compile the schemas, so I guess you can make changes accordingly.